src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -163,83 +366,24 @@ def create(self, dataset_name, options={}):
         -------
         json
         """
-
-        if options.get("like") is None:
-            if options.get("primary") is None or options.get("lrecl") is None:
-                self.logger.error("If 'like' is not specified, you must specify 'primary' or 'lrecl'.")
-                raise ValueError("If 'like' is not specified, you must specify 'primary' or 'lrecl'.")
-
-            for opt in (
-                "volser",
-                "unit",
-                "dsorg",
-                "alcunit",
-                "primary",
-                "secondary",
-                "dirblk",
-                "avgblk",
-                "recfm",
-                "blksize",
-                "lrecl",
-                "storclass",
-                "mgntclass",
-                "dataclass",
-                "dsntype",
-                "like",
-            ):
-                if opt == "dsorg":
-                    if options.get(opt) is not None and options[opt] not in ("PO", "PS"):
-                        self.logger.error(f"{opt} is not 'PO' or 'PS'.")
-                        raise KeyError
-
-                elif opt == "alcunit":
-                    if options.get(opt) is None:
-                        options[opt] = "TRK"
-                    else:
-                        if options[opt] not in ("CYL", "TRK"):
-                            self.logger.error(f"{opt} is not 'CYL' or 'TRK'.")
-                            raise KeyError
-
-                elif opt == "primary":
-                    if options.get(opt) is not None:
-                        if options["primary"] > 16777215:
-                            self.logger.error("Specified value exceeds limit.")
+        if options:
```

Comment:
If the `options` object is empty our validation should fail. I think we could define `options = options or {}` at the top of this function, to ensure that the object is not None, and then this `if` condition could be removed.